dask_geopandas/geohash.py

```diff
@@ -31,6 +33,15 @@ def _geohash(gdf, as_string, precision):
     type : pandas.Series
         Series containing geohash
     """
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore", "GeoSeries.isna() previously returned True", UserWarning
+        )
```

Comment:
I am also going to remove this from geopandas. It has been there for ages.

Reply:
Yes, that's a good idea